app/main.py

```diff
@@ -1,13 +1,5 @@
-# from models import Actor
```

Comment:
Do not include db files to the pull request as mentioned in the [checklist](https://github.com/loplicat/py-actor-manager/blob/develop/checklist.md#dont-push-db-files).